### PR TITLE
From code camp, add note to NXmx that file_name and file_time are recommended

### PR DESCRIPTION
**Pull request by phyy-nx**
See also nexusformat/cnxvalidate#24

### CODE REVIEW

File: applications/NXmx.nxdl.xml [hunk 1]
```diff
@@ -60,6 +60,12 @@
 		</doc>
 
 		<group type="NXentry">
+
+			<doc>
+				Note, it is recommended that file_name and file_time are included as fields
```
Comment (prjemian) [inline applications/NXmx.nxdl.xml]: **Not** fields, add them as attributes.

File: applications/NXmx.nxdl.xml [hunk 2]
```diff
@@ -60,6 +60,12 @@
...
+				at the root of a file that includes NXmx. See :ref:`NXroot`.
```
Comment (prjemian) [inline applications/NXmx.nxdl.xml]: :ref:`NXmx`

**Comment by prjemian** [comment]
I'm marking this for the milestone this week.  If it's not resolved by COB Thursday, I'll move it to the next milestone.

**Comment by phyy-nx** [comment]
Addressed comments.  Thanks.  I'll merge this on Thursday unless there are objections.

File: applications/NXmx.nxdl.xml [hunk 3]
```diff
@@ -62,8 +62,9 @@
 		<group type="NXentry">
 
 			<doc>
-				Note, it is recommended that file_name and file_time are included as fields
-				at the root of a file that includes NXmx. See :ref:`NXroot`.
+				Note, it is recommended that file_name and file_time are included as
```
Comment (prjemian) [inline applications/NXmx.nxdl.xml]: ``file_name`` and ``file_time``

**Comment by phyy-nx** [comment]
@prjemian ok to merge?

**Pull request merged**